From 4d71c59047dd0193add263b1780393d20e80e90d Mon Sep 17 00:00:00 2001
From: Jun Zhang <jun.zhang@amlogic.com>
Date: Wed, 14 Apr 2021 19:55:50 +0800
Subject: [PATCH 5/8] RDK: fix modpost not compiled issue [1/1]

PD#SWPL-

Problem:
 modpost not compiled during do_make_scripts

Solution:
 force add subdir of mod

Verify:
local

Change-Id: I6e9d177229b85a7ec025edd2062961b427166a7f
Signed-off-by: Jun Zhang <jun.zhang@amlogic.com>
---
 scripts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/Makefile b/scripts/Makefile
index 3e86b300f5a1..76131f2da435 100644
--- a/scripts/Makefile
+++ b/scripts/Makefile
@@ -34,6 +34,7 @@ hostprogs-y += unifdef
 subdir-$(CONFIG_GCC_PLUGINS) += gcc-plugins
 subdir-$(CONFIG_MODVERSIONS) += genksyms
 subdir-$(CONFIG_SECURITY_SELINUX) += selinux
+subdir-y += mod
 
 # Let clean descend into subdirs
 subdir-	+= basic dtc gdb kconfig mod
-- 
2.29.2

